uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -99,6 +99,33 @@ public String acquireToken(@NonNull final String username,
         return result.getText();
     }
 
+    // click on button acquire token interactive
+    public void acquireTokenWithoutFetchingToken(@NonNull final String username,
+                                                 @NonNull final String password,
+                                                 @NonNull final PromptHandlerParameters promptHandlerParameters,
+                                                 @NonNull final boolean shouldHandlePrompt) throws UiObjectNotFoundException, InterruptedException {
+
+        final UiObject acquireTokenButton = UiAutomatorUtils.obtainUiObjectWithResourceId("com.msft.identity.client.sample.local:id/btn_acquiretoken");
+        scrollToElement(acquireTokenButton);
+        acquireTokenButton.click();
+
+        // handle prompt if needed
+        if (shouldHandlePrompt) {
+            try {
+                final UiObject emailField = UiAutomatorUtils.obtainUiObjectWithTextAndClassType(
+                        "", EditText.class);
```

Comment:
Is there no resource id on this e-mail field?

Reply:
apparently not, i ended up not trying to use msalTestApp as we have some other changes needed to support it, this code was added by LTW folks. I guess we don't set a resource id for this field.